tests/bin/demo-addition.test.ts

```diff
@@ -0,0 +1,28 @@
+// this test fails -> attempting to check that the output of console.log is correct
```

Comment:
You have to use jest to spyon or mock the process.stdout.

Note that the command isn't using `console.log` to output data.

It's using `process.stdout` or `process.stderr`.

To simplify doing this, see: https://github.com/EpicEric/jest-mock-process